library/spdm_responder_lib/libspdm_rsp_chunk_send_ack.c

```diff
@@ -227,17 +227,14 @@ libspdm_return_t libspdm_get_response_chunk_send(libspdm_context_t *spdm_context
 
     if (libspdm_get_connection_version(spdm_context) < SPDM_MESSAGE_VERSION_14) {
         spdm_response->chunk_seq_no = spdm_request->chunk_seq_no;
-
-        chunk_response = (uint8_t*) (spdm_response + 1);
-        chunk_response_size = *response_size - sizeof(spdm_chunk_send_ack_response_t);
     } else {
         spdm_response_14 = response;
         spdm_response_14->chunk_seq_no = spdm_request->chunk_seq_no;
```

Comment:
I think this should be `spdm_request_14->chunk_seq_no`.

In general, I think we should define `uint32_t chunk_seq_no`.
In the beginning, 
```
if (1.4) {
    chunk_seq_no = spdm_request_14->chunk_seq_no;
} else {
    chunk_seq_no = spdm_request->chunk_seq_no;
}
```

Then always use `chunk_seq_no` in rest of the places.